nucleus/utils.py

```diff
@@ -41,6 +41,12 @@
 )
 from .scene import LidarScene
 
+STRING_REPLACEMENTS = {
+    "\\\\n": "\n",
+    "\\\\t": "\t",
+    '\\\\"': '"',
+}
```

Comment:
prob could just replace \\\\ with \ to get the desired effect not sure when \\\\ would come up on its own